product_code_unique/readme/CONTRIBUTORS.rst

```diff
@@ -0,0 +1,9 @@
+* Antonio Yamuta <ayamuta@opensourceintegrators.com>
+* Raf Ven <raf.ven@dynapps.be>
+
+Funders
```

Comment:
Remove this part, as they are already creators.

Reply:
> Remove this part, as they are already creators.

Do you mean the Funders parts?

Reply:
Yes